_posts/2023-05-15-update-in-steering-committee.md

```diff
@@ -0,0 +1,9 @@
+---
+title: "Changes in the steering committee"
+author: bziliani
+summary: Juan Wajnerman is stepping down from his position, with Beta Ziliani taking his place.
+---
+
+Juan Wajnerman has been involved with Crystal since its very beginning, and has served as one of the three Steering Committee members (the other two being Ary Borenzweig and Brian Cardiff). However, he decided to move on with his personal projects and leave this position. I am honoured to be selected as Juan's replacement.
```

Comment:
```suggestion
Juan Wajnerman has been involved with Crystal since its very beginning, and has served as one of the three Steering Committee members (the other two being Ary Borenzweig and Brian Cardiff). However, he decided to leave this position and move on with his personal projects. I am honoured to be selected as Juan's replacement.
```